src/components/charts/ReferencesSearch.tsx

```diff
@@ -0,0 +1,86 @@
+import _ from 'lodash'
+import React from 'react'
+import { Search } from 'semantic-ui-react'
+import { AirtableRecord } from '../../types';
+import Translate from "../../utils/translate/translateService";
+                                                        
+const initialState = {
+  loading: false,
+  results: [],
+  value: '',
+}
+
+const MAX_SUGGESTIONS = 5
+
+function searchReducer(state : any, action : any) {
```

Comment:
good idea! maybe we want to abstract the reducer logic into its own file

Reply:
I can do this 👍